.github/workflows/checks.yml

```diff
@@ -0,0 +1,45 @@
+name: Check sctipt runners
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    name: Run lint sctipt
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run lint
+        run: npm run lint
+  test:
+    name: Run test sctipt
```

Comment:
Run tests

Reply:
`test` is the name of the script in the package.json. See [here](https://github.com/tolstoyevsky/cusdeb-web-client/blob/master/package.json#L12).